packages/rdbg/rdbg.1.0/opam

```diff
@@ -0,0 +1,24 @@
+opam-version: "1.2"
+maintainer: "erwan.jahier@imag.fr"
+bug-reports: "erwan.jahier@imag.fr"
+authors: [ "Erwan Jahier" ]
+license: "GPL-3"
+homepage: "http://rdbg.forge.imag.fr/"
+dev-repo: "https://forge.imag.fr/anonscm/git/rdbg/rdbg.git"
+build: [
+  ["oasis" "setup"]
+  ["ocaml" "setup.ml" "-configure" "--prefix" prefix]
+  ["ocaml" "setup.ml" "-build"]
+]
+install: ["ocaml" "setup.ml" "-install"]
+remove: [
+  ["ocamlfind" "remove" "rdbg-plugin"]
+]
+build-doc: [ "ocaml" "setup.ml" "-doc" ]
+depends: [
+  "base-unix"
+  ("camlp4" {= "4.03.0"} | "camlp4" {>= "4.02.1+system"} | "camlp4" {= "4.02.1+2"} | "camlp4" {= "4.02.1+1"} | "camlp4" {= "4.02.1"} | "camlp4" {= "4.02.0-1modular-implicits"} | "camlp4" {= "4.02.0+system"} | "camlp4" {= "4.02.0+2"} | "camlp4" {= "4.02.0+1"} | "camlp4" {= "4.02.0"} | "camlp4" {= "4.01.0"})
```

Comment:
I believe you can simply edit the opam file and then `opam publish` again and it will update this PR (I believe it does a force push but I could be wrong). Also, it would be nice if you could file an issue in the `oasis2opam` tracker about this. Thank you for your contribution!